spring-cloud-app-broker-deployer/src/main/java/org/springframework/cloud/appbroker/deployer/AppDeployer.java

```diff
@@ -32,6 +32,10 @@
 		return Mono.empty();
 	}
 
+	default Mono<GetApplicationResponse> get(GetApplicationRequest request) {
```

Comment:
I think maybe `getApplication` is better.

Reply:
I initially have the name as `getApplication` as you suggested, but changed it afterwards becaulse all other application related methods in the context don't have `application` in the names, e.g `deploy`, `update`. I agree that `getApplication` is better, should we append `application` for all of them?

Reply:
I would go with `get`, although I see @royclarkson's point I would rather be consistent and not cause a breaking change.

Reply:
yep, agreed. I wasn't looking at the whole file, so missed the other methods are the same. Thanks.